packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1698,12 +1725,17 @@ export const components = ({
           },
         }),
         input: {
-          boxSizing: "border-box",
-          height: "auto",
+          // Set total height to 40px, factoring in borders on outer container
+          height: `calc(${odysseyTokens.Spacing4} - (${odysseyTokens.BorderWidthMain} * 2))`,
```

Comment:
Can we change the comment to not specify a pixel value and instead explain why the calculation is the way it is? Because if we change token spacing, this won't be 40px anymore.